src/lib/tooltip/tooltip.spec.ts

```diff
@@ -38,25 +39,87 @@ describe('MdTooltip', () => {
       tooltipDirective = buttonDebugElement.injector.get(MdTooltip);
     });
 
-    it('should show/hide on mouse enter/leave', () => {
-      expect(tooltipDirective.visible).toBeFalsy();
+    it('should show and hide the tooltip', fakeAsync(() => {
+      expect(tooltipDirective._tooltipInstance).toBeUndefined();
+
+      tooltipDirective.show();
+      expect(tooltipDirective._isTooltipVisible()).toBe(true);
+
+      fixture.detectChanges();
+      expect(overlayContainerElement.textContent).toContain(initialTooltipMessage);
+
+      // After hide called, a timeout delay is created that will to hide the tooltip.
+      tooltipDirective.hide();
+      expect(tooltipDirective._isTooltipVisible()).toBe(true);
+
+      // After the tooltip delay elapses, expect that the tooltip is not visible.
+      tick(TOOLTIP_HIDE_DELAY);
+      fixture.detectChanges();
+      expect(tooltipDirective._isTooltipVisible()).toBe(false);
+
+      // On animation complete, should expect that the tooltip has been detached.
+      fixture.whenStable().then(() => {
```

Comment:
You shouldn't need `whenStable` with `fakeAsync`, since you're in control of ticking the clock. 
(unless there's something special going on with ngAnimate that I don't know about)